src/services/state-persistor/state-persistor-features.service.ts

```diff
@@ -35,6 +36,8 @@ class StatePersistorFeaturesService implements StatePersistorService {
 
   restore() {
     const { drawnFeatures } = storeToRefs(useDrawStore())
+    // initialise map listeners for feature selection
+    useDrawSelect()
```

Comment:
It feels a little strange to call a composable in a state persistor, but I agree that it makes sense to concentrate this logic in a composable and don't have a better suggestion neither.